src/win/win_progsett.c

```diff
@@ -107,6 +184,10 @@ progsett_settings_save(void)
     /* Language */
     set_language(temp_language);
 
+	/* Iconset */
+	strcpy(icon_set, temp_icon_set);
+	win_load_icon_set(hinstance);
+
```

Comment:
Indentation + wrong parameters for `win_load_icon_set`

Reply:
Thank you for the checks and comments, I've applied a fix for these parts of the code too, according to indentation.

If I can mention one thing, where this feature will have a really practical use in the near future, is developing the new icon set (in the same style of the main icon). At the Discord server I've demonstrated that it can provide a very rapid developing environment for this task, where it's not necessary to recompile 86Box to see how the new changes will look in the program. It allows to test quickly the icons, and reload them even after the slightest changes, and only compile them into the program itself, when all of them are absolutely ready.